README.md

```diff
@@ -48,4 +48,4 @@ the exact (byte-to-byte) same binaries which are published to https://releases.h
 
 ### Why doesn't HashiCorp maintain formulae in the main tap?
```

Comment:
I would replace `main tap` with `homebrew-core` to be more clear about what we're talking about here

Reply:
Can you check my edit and let me know what you think?

Reply:
Good with me